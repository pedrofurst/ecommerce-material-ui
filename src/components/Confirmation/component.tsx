import { Typography } from '@material-ui/core';
import useStyles from './styles';

type ConfirmationPropsType = {};

function Confirmation(props: ConfirmationPropsType) {
  const classes = useStyles();
  return (
    <div className={classes.confirmationContainer}>
      <Typography variant="h5" gutterBottom>
        Thank you for your order.
      </Typography>
      <Typography variant="subtitle1">
        Your order number is #2001539. We have emailed your order confirmation,
        and will send you an update when your order has shipped.
      </Typography>
    </div>
  );
}

export default Confirmation;
