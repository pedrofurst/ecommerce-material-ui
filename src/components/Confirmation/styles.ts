import { makeStyles } from '@material-ui/core/styles';

export default makeStyles({
  confirmationContainer: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    paddingTop: 40,
  },
});
