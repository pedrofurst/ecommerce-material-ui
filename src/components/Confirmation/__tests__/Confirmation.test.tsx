import renderer from 'react-test-renderer';
import Confirmation from '../component';

describe('<Confirmation />', () => {
  it('Should have confirmation message', () => {
    const component = renderer.create(<Confirmation />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
