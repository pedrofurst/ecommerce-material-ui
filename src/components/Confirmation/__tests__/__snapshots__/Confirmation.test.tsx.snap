// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Confirmation /> Should have confirmation message 1`] = `
<div
  className="makeStyles-confirmationContainer-1"
>
  <h5
    className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
  >
    Thank you for your order.
  </h5>
  <h6
    className="MuiTypography-root MuiTypography-subtitle1"
  >
    Your order number is #2001539. We have emailed your order confirmation, and will send you an update when your order has shipped.
  </h6>
</div>
`;
